src/esssans/types.py

```diff
@@ -153,15 +153,25 @@ class NeXusMonitorName(sciline.Scope[MonitorType, str], str):
     """Name of Incident|Transmission monitor in NeXus file"""
 
 
-class FileList(sciline.Scope[RunType, list], list):
-    """Filenames of BackgroundRun|EmptyBeamRun|SampleRun"""
+PixelMaskFilename = NewType('PixelMaskFilename', str)
```

Comment:
Has it been decided that ESS will use ISIS/Mantid pixel mask files? If not, I think this (and related code) should remain in the `esssans.isis` module.

Reply:
I don't think it's been decided, but I imagine that loading a mask from file would be one of the way the users would want to mask pixels.
I think we do not need to use the Mantid format though, which is why I left the XML details in the `isis` module.
We could use scipp's hdf5 format to store masks, but you'd still have a `PixelMaskFilename` and a `PixelMask`?